docs/node/run-your-node/prerequisites/stake-requirements.md

```diff
@@ -53,14 +56,19 @@ the [**max_validators** consensus parameter][**max_validators** consensus parame
 [faucet-testnet]: https://faucet.testnet.oasis.io/
 [discord]: https://oasis.io/discord
 
-## ParaTime
+## ParaTime Compute Node
 
-To run a ParaTime compute node, the minimum you need to stake currently is:
+To run a ParaTime compute node, on Mainnet you will need to have a minimum
+stake which currently is:
 
-|            | Mainnet     | Testnet  |
-| ---------- | ----------- | -------- |
-| Sapphire   | 5,000,100 ROSE | 100 TEST |
-| Emerald    | 5,000,100 ROSE | 100 TEST |
-| Cipher     | 100 ROSE    | 100 TEST |
+|                       | Mainnet        | Testnet  |
+| --------------------- | -------------- | -------- |
+| Be in validator set   | Yes            | No       |
```

Comment:
This is somewhat out of place for this table? Maybe just mention it in text instead or make it per-runtime (which it is, as each runtime can define the requirements).